src/mixins/room-mixin.ts

```diff
@@ -61,6 +61,8 @@ const roomMixin = <MixinBase extends typeof PuppetSkeleton & ContactMixin & Room
       */
     abstract roomAnnounce (roomId: string)               : Promise<string>
     abstract roomAnnounce (roomId: string, text: string) : Promise<void>
+    abstract roomAnnounce (roomId: string, text: string, isAnnounce: boolean) : Promise<string | void>
+    abstract roomAnnounce (roomId: string, text: string, isAnnounce: false, mentionList: string[]) : Promise<string>
```

Comment:
I think the `isAnnounce` is unclear for what its meaning is. I remember you mentioned the `pin` I think it's a better name, how do you think about let's change this name from `isAnnounce` to `pin`?

And I forgot why we need to add `mentionList` here, can you explain it briefly? 

Thanks!

Reply:
This PR is outdated and should be closed.
As we talked before, we should use '@${RoomId}' to send @all messages.